### PR TITLE
[+] Adds group member update route and hooks up to edit/delete endpoints

### PR DESCRIPTION
**Pull request by andy-smith-cds**

### CODE REVIEW

File: futurenhs.app/pages/groups/[groupId]/members/[memberId]/update/index.page.tsx [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+                                requestMethod === requestMethods.POST
+                            ) {
+
+                                const headers =
+                                getStandardServiceHeaders({
+                                    csrfToken,
+                                    etag,
+                                });
+
+                                if (currentValues.body?.['_form-id'] === updatedRolesForm.id) {
```
Comment (ReallySmall) [inline futurenhs.app/pages/groups/[groupId]/members/[memberId]/update/index.page.tsx:125]: This feels like a thing to abstract/ avoid typos using a more generic selector

e.g. isFormType(currentValues, updatedRolesForm.id)

Comment (andy-smith-cds) [reply futurenhs.app/pages/groups/[groupId]/members/[memberId]/update/index.page.tsx]: Made a helper for this and now checking if (isRoleForm) / (isDeleteForm)

**Comment by Matt-Beldon-CDS** [comment]
Checked this out and it all works fine locally

**Comment by andy-smith-cds** [comment]
Cheers Matt. Happy to merge or awaiting approval from Tim?

**Review by Matt-Beldon-CDS — APPROVED** [review]
> Cheers Matt. Happy to merge or awaiting approval from Tim?

Happy to merge!

**Pull request merged**